src/abaqus/Region/RegionAssembly.py

```diff
@@ -193,17 +192,17 @@ def Set(
         nodes: Optional[Sequence[MeshNode]] = None,
         elements: Optional[Sequence[MeshElement]] = None,
         region: Optional[Region] = None,
-        vertices: Optional[Sequence[Vertex]] = None,
+        vertices: Optional[Sequence[ConstrainedSketchVertex]] = None,
         edges: Optional[Sequence[Edge]] = None,
         faces: Optional[Sequence[Face]] = None,
         cells: Optional[Sequence[Cell]] = None,
-        xVertices: Optional[Sequence[Vertex]] = None,
+        xVertices: Optional[Sequence[ConstrainedSketchVertex]] = None,
         xEdges: Optional[Sequence[Edge]] = None,
         xFaces: Optional[Sequence[Face]] = None,
         referencePoints: Sequence[ReferencePoint] = (),
-        skinFaces: tuple = ...,
-        skinEdges: tuple = ...,
-        stringerEdges: tuple = ...,
+        skinFaces: Tuple[Tuple[str, Face], ...] = ...,
+        skinEdges: Tuple[Tuple[str, Edge], ...] = ...,
+        stringerEdges: Tuple[Tuple[str, Edge], ...] = ...,
```

Comment:
Should this be something like `Tuple[Tuple[str, Sequence[Face]], ...]`?

Reply:
Yes, you're right

Reply:
Also changed in [2879e8d](https://github.com/haiiliin/abqpy/pull/2872/commits/2879e8d4fbcf5f414bb6ab0d011f997e418dd61d)